# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1354656852 18000
#      Tue Dec 04 16:34:12 2012 -0500
# Node ID e56e8f311341c7b781dca4c87cfdc627f1c501cd
# Parent  1090b62af49d243bef3896f914e78524307253cb
Don't link liboctinterp with $(GRAPHICS_LIBS).

* libinterp/link-deps (LIBOCTNITERP_LINK_DEPS): Remove $(GRAPHCS_LIBS)
from the list.
(LIBOCTINTERP_LINK_OPTS): Remove $(GRAPHICS_LDFLAGS) from the list.

diff --git a/libinterp/link-deps.mk b/libinterp/link-deps.mk
--- a/libinterp/link-deps.mk
+++ b/libinterp/link-deps.mk
@@ -2,31 +2,29 @@ include $(top_srcdir)/liboctave/link-dep
 
 if AMCOND_ENABLE_DYNAMIC_LINKING
   LIBOCTINTERP_LINK_DEPS =
 else
   LIBOCTINTERP_LINK_DEPS = $(DLDFCN_LIBS)
 endif
 
 LIBOCTINTERP_LINK_DEPS += \
-  $(GRAPHICS_LIBS) \
   $(FT2_LIBS) \
   $(HDF5_LIBS) \
   $(Z_LIBS) \
   $(FFTW_XLIBS) \
   $(REGEX_LIBS) \
   $(OPENGL_LIBS) \
   $(X11_LIBS) \
   $(CARBON_LIBS) \
   $(LLVM_LIBS) \
   $(JAVA_LIBS) \
   $(LAPACK_LIBS)
 
 LIBOCTINTERP_LINK_OPTS = \
-  $(GRAPHICS_LDFLAGS) \
   $(FT2_LDFLAGS) \
   $(HDF5_LDFLAGS) \
   $(Z_LDFLAGS) \
   $(REGEX_LDFLAGS) \
   $(FFTW_XLDFLAGS) \
   $(LLVM_LDFLAGS)
 
 OCT_LINK_DEPS =
